rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/util/FormKeyTextExtractionUtilTest.java

```diff
@@ -0,0 +1,19 @@
+package io.ona.rdt.robolectric.util;
+
+import org.json.JSONException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Map;
+
+import io.ona.rdt.robolectric.RobolectricTest;
+import io.ona.rdt.util.FormKeyTextExtractionUtil;
+
+public class FormKeyTextExtractionUtilTest extends RobolectricTest {
+
+    @Test
+    public void testGetFormWidgetKeyToTextMap() throws JSONException {
+        Map<String, String> data = FormKeyTextExtractionUtil.getFormWidgetKeyToTextMap();
+        Assert.assertFalse(data.isEmpty());
```

Comment:
Could we add more verification that data extracted is what is expected?

Reply:
added size verification

Reply:
Verify that the actual data matches what is expected for a given set of forms.

Reply:
fixed